### PR TITLE
test: add test for use_nvmrc in toolchain creation

### PR DESCRIPTION
**Pull request by jstuder-gh**
Leverages existing `nodejs_host` test.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?
Adds e2e test for use_nvmrc in node toolchain.

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ X] Other... Please describe: Test change

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ X] No

### CODE REVIEW

**Comment by jstuder-gh** [comment]
Is it appropriate modify the existing nodejs_host test for this purpose? Was the intent of that test primarily to verify that the default "nodejs_host", "npm", and "yarn" repositories are created (as opposed to the primary purpose being the use of the proper node version in various operations)? If so, I butchered that test.

Would it be more appropriate to break this out into a separate e2e test? One explicitly for multiple node toolchains/repos and leave this one for testing creation of the nodejs_host repository?

File: e2e/nodejs_host/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {},
   "scripts": {
-    "postinstall": "node -e \"require('assert').equal(process.version, 'v12.12.0')\"",
+    "postinstall": "...",
```
Comment (alexeagle) [inline e2e/nodejs_host/package.json:4]: nit: I'd use this placeholder value more explicitly, it's not clear that `...` gets replaced. Maybe "placeholder" or "replaced during yarn_install"

**Review by alexeagle — APPROVED** [review]
looks good to me. IMO it's nice to keep the CI workload lower/faster by re-using existing e2e where possible, and this just generalizes a bit.

File: e2e/nodejs_host/package.json (hunk 1 above)
Comment (jstuder-gh) [reply e2e/nodejs_host/package.json:4]: Yeah you're right, that's much more clear. I'll change and re-push.

**Review by alexeagle — APPROVED** [review]
🙏🏻

**Pull request merged**